src/test/resolveNodeModuleToDenoModule/getValidImportUrl/test12.ts

```diff
@@ -0,0 +1,35 @@
+import { ModuleAddress } from "../../../lib/types/ModuleAddress";
+import { assert } from "tsafe";
+import { getValidImportUrlFactory } from "../../../lib/resolveNodeModuleToDenoModule";
+
+(async () => {
+    const userOrOrg = "GervinFung";
+    const moduleAddress: ModuleAddress.GitHubRepo = {
+        "type": "GITHUB REPO",
+        userOrOrg,
+        "repositoryName": "my_dummy_npm_and_deno_module",
+        "branch": "master"
+    };
+
+    {
+        const getValidImportUrlFactoryResult = await getValidImportUrlFactory({
+            moduleAddress,
+            "desc": "NOT LISTED AS A DEPENDENCY (PROBABLY NODE BUILTIN)"
+        });
+
+        assert(getValidImportUrlFactoryResult.couldConnect === true);
+
+        assert(getValidImportUrlFactoryResult.versionFallbackWarning === undefined);
+
+        assert(
+            (await getValidImportUrlFactoryResult.getValidImportUrl({ "target": "DEFAULT EXPORT" })) ===
+                `https://raw.githubusercontent.com/${userOrOrg}/my_dummy_npm_and_deno_module/master/deno_dist/mod.ts`
+        );
+        assert(
+            (await getValidImportUrlFactoryResult.getValidImportUrl({ "target": "SPECIFIC FILE", "specificImportPath": "dist/lib/Cat" })) ===
+                `https://raw.githubusercontent.com/${userOrOrg}/my_dummy_npm_and_deno_module/master/deno_dist/lib/Cat.ts`
+        );
+    }
+
+    console.log("PASS");
+})();
```

Comment:
This test is a very good idea!  
A couple of thing I would like to be addressed:  
- Could you create an other fork and an other test (13) where your config file would have an explicite [a out parameter](https://github.com/mxxii/leac/blob/db4c80ffcb075ef5ef197205d9d743c349d5e4fc/package.json#L86-L88) to make sure that your file is wel parsed.  
- [Create a GitHub release](https://user-images.githubusercontent.com/6702424/192179447-33ebadf8-ef45-45e5-9291-43b18213eae6.mov) for your forks, and use:  

```javascript
            "desc": "MATCH VERSION INSTALLED IN NODE_MODULES",
            "version": "v1.0.0"
```
instead of `NOT LISTED AS A DEPENDENCY (PROBABLY NODE BUILTIN)`, it's a bit confusing, it make it hard to understand what we are testing.  

Thank you very much!